packages/next/build/babel/preset.ts

```diff
@@ -55,37 +57,46 @@ function supportsStaticESM(caller: any) {
 
 module.exports = (api: any, options: NextBabelPresetOptions = {}): BabelPreset => {
   const supportsESM = api.caller(supportsStaticESM)
+
   const presetEnvConfig = {
     // In the test environment `modules` is often needed to be set to true, babel figures that out by itself using the `'auto'` option
     // In production/development this option is set to `false` so that webpack can handle import/export with tree-shaking
     modules: 'auto',
-    exclude: ['transform-typeof-symbol'],
+    exclude: [
+      // used to improve react's performance. see: https://github.com/zeit/next.js/pull/6812
```

Comment:
a comment to prevent people who look at this file from investigating why this exclude is here